### DISCUSSION TITLE
Separate volume settings for UI sound effects and for hitsounds

### DISCUSSION THREAD

**Discussion by ItsCollector** [ppy/osu#30911] [Ideas] [answered]
Particularly in osu!mania, most people do not use hitsounds by having effects volume on zero as it is a distraction more than anything (unless the map has custom hitsounds to match the song, which is a small amount of maps).

Having effects sounds off to ensure we don't hear hitsounds also means we cannot hear the sounds that all the UI makes when not playing a beatmap. 

Having separate volume settings for these means palayers who don't want to hear hitsounds can have it muted, while having a separate volume setting for the UI sound effects (which are very satisfying to hear, might I add).

**Comment by WalmartWhiteCat** [comment]
I have the same want as a Standard player. I hate hitsounds but the menu sounds are awesome. It’s so jank having to switch effects on and off all the time. I would just replace the hitsounds in my skin to be silent, but I’m on Linux so that’s not really possible since I can’t access the skin folder at all

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/discussions/25068
https://github.com/ppy/osu/pull/28733
